nemo/collections/nlp/data/machine_translation/machine_translation_dataset.py

```diff
@@ -36,8 +35,14 @@
 
 @dataclass
 class TranslationDataConfig:
-    src_file_name: str = MISSING
-    tgt_file_name: str = MISSING
+    src_file_name: Optional[str] = None
+    tgt_file_name: Optional[str] = None
+    use_tarred_dataset: bool = False
+    tar_files: Optional[str] = None
+    metadata_file: Optional[str] = None
+    lines_per_dataset_fragment: Optional[int] = 1000000
```

Comment:
what will happen if the total dataset size is less than this number?

Reply:
Added a warning if too few tar files found and also suggested remedies: decrease `num_batches_per_tarfile`, decrease `num_tokens_per_batch`, or use `shard_strategy='replicate'`.